recipes/gsee/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "gsee" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "a20d24f85fb77276713ccd6a25df884a2de4195909532b7f36f91a3ec50ed8ea" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy >=1.15
+  run:
+    - python
+    - joblib >=0.12
```

In relation to the above comment, you probably need a pinning for `numpy` here:
```suggestion
    - {{ pin_compatible('numpy') }}
    - joblib >=0.12
```